lib/node_modules/@stdlib/random/base/mt19937/docs/repl.txt

```diff
@@ -0,0 +1,96 @@
+
+{{alias}}()
+    Returns a pseudorandom integer on the interval `[1, 2147483646]`.
+
+    This pseudorandom number generator (PRNG) is a 32-bit Mersenne Twister
+    pseudorandom number generator.
+
+    The generator has a period of `2^{19937} - 1`.
+
+    Returns
+    -------
+    r: integer
+        Pseudorandom number.
+
+    Examples
+    --------
+    > var r = {{alias}}();
+
+
+{{alias}}.normalized()
+    Returns a pseudorandom number on the interval `[0,1)`.
+
+    Returns
+    -------
+    r: number
+        Pseudorandom number.
+
+    Examples
+    --------
+    > var r = {{alias}}.normalized();
+
+
+{{alias}}.factory( [seed] )
+    Returns a 32-bit Mersenne Twister pseudorandom number generator.
+
+    Parameters
+    ----------
+    seed: integer (optional)
+        Pseudorandom number generator seed. Must be on the interval
```

Comment:
The range needs to be updated.

Reply:
Did this, not sure why this is not "outdated".